README.md

```diff
@@ -185,27 +185,43 @@ The following options can be set for any particular page. While the default opti
   * `name` - If the name of the thing that the page represents is different from the page title. (i.e.: "Frank's Café" vs "Welcome to Frank's Café")
   * `type` - The type of things that the page represents. This must be a [Schema.org type](http://schema.org/docs/schemas.html), and will probably usually be something like [`BlogPosting`](http://schema.org/BlogPosting), [`NewsArticle`](http://schema.org/NewsArticle), [`Person`](http://schema.org/Person), [`Organization`](http://schema.org/Organization), etc.
   * `links` - An array of other URLs that represent the same thing that this page represents. For instance, Jane's bio page might include links to Jane's GitHub and Twitter profiles.
+  * `dateModified` - An override for the `dateModified` field in the JSON-LD output. Useful when the file timestamp does not match the true time that the content was modified. 
+  * `author` - The `author` field will match the JSON-LD author output where applicable.
```

Comment:
It doesn't differ. It was documentation letting the user know that the author field was now being used in the JSON-LD. I can see how its placement would imply that it is an additional entry. I will remove this because the author field is already mentioned as a global property earlier in the documentation.